src/components/account/notification-settings/NotificationSettingsFormContent.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import Link from 'next/link';
+import { useTranslations } from 'next-intl';
+import type { UseFormRegister } from 'react-hook-form';
+
+import type { TypeNotificationSettings } from '@/__generated__/data-contracts';
+import Button from '@/ui/button/button';
+import ButtonGroup from '@/ui/button-group/button-group';
+import Checkboxes from '@/ui/checkboxes/checkboxes';
+
+function Option({ name, label, register }: { name: keyof TypeNotificationSettings; label: string; register: UseFormRegister<TypeNotificationSettings> }) {
+  return (
+    <Checkboxes
+      name={name}
+      legend={label}
+      items={[
+        {
+          id: `${name}_email`,
+          children: 'Email',
+          value: 'EMAIL',
+          ...register(name),
+        },
+        {
+          id: `${name}_sms`,
+          children: 'Text',
+          value: 'SMS',
+          ...register(name),
+        },
+      ]}
+    />
+  );
+}
+
+type NotificationSettingsFormContentProps = {
+  isSubmitting: boolean;
+  register: UseFormRegister<TypeNotificationSettings>;
+};
+
+const NotificationSettingsFormContent = ({ isSubmitting, register }: NotificationSettingsFormContentProps) => {
+  const t = useTranslations('Forms.Notification_settings');
+  const tCommon = useTranslations('Common');
+
+  return (
+    <div>
+      <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
```

Comment:
Maybe there is place to create component `<HorizontalLine />` ?
I see we use it across application

Reply:
I think we have it only in this one place